### ISSUE TITLE
lazy reverse(a; dims=n) with keyword dimension

### ISSUE THREAD

**Issue by felixcremer** [meggart/DiskArrays.jl#113]
It would be good to dispatch on the DiskArray implementation for the lazy reverse when one specifies the dimension by a keyword argument. 
It dispatches directly if the dimension is the second positional argument, but the keyword argument is dispatched into the default Base implementation.

```julia
a:: DiskArray

julia> reverse(a, dims=1) # Non lazy
10000×10000 Matrix{UInt8}:
 0x00  0x00  …  0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00  …  0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00     0x00  0x00
    ⋮        ⋱        
 0x00  0x00     0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00  …  0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00     0x00  0x00
 0x00  0x00     0x00  0x00

julia> reverse(a, 1) # Lazy 
Disk Array with size 10000 x 10000
```

**Comment by rafaqz** [comment]
Just needs to add the keyword version here:
https://github.com/meggart/DiskArrays.jl/blob/main/src/array.jl#L31

**Comment by rafaqz** [comment]
This was fixed by #150
